ntbs-service/Pages/Alerts/TransferRequestAction.cshtml.cs

```diff
@@ -92,6 +92,22 @@ public async Task<IActionResult> OnPostAsync()
                 return Partial("_AcceptedTransferConfirmation", this);
             }
             
+            var transferRejectedAlert = new TransferRejectedAlert()

```

Comment:
This method is getting quite chunky (`OnPostAsync`) reckon we could split it up?